### ISSUE TITLE
Warning: The PGconn, PGresult, and PGError constants are deprecated

### ISSUE THREAD

**Issue by KrzysztofMadejski** [consul/consul#3108]
### User Story

As an admin I want to setup the database.

### Description

While running `rake db:create` (from Docker) I'm getting warning
```
The PGconn, PGresult, and PGError constants are deprecated, and will be
removed as of version 1.0.

You should use PG::Connection, PG::Result, and PG::Error instead, respectively.
```

### Expected Behavior
No warning. Updated code.

**Comment by jaflutz** [comment]
Solutions on the web point changing from gem 'pg', '~> 0.21.0' to gem 'pg', '~> 0.20.0'. It worked for me, but I don't know if it is the appropriate solution.

**Comment by microweb10** [comment]
We are planning to upgrade to Rails 5, and therefore to upgrade some gems too. Then this warning should be gone. 

Also it's worth mentioning that it's just a warning. Everything should work fine if you use the gem version we have in the file `Gemfile` https://github.com/consul/consul/blob/master/Gemfile#L39

**Comment by javierm** [comment]
I got so used to this warning I didn't realize it was gone :smile:. I guess it was solved by #3414.
